src/sadhana/CommonElementsBitweenArray.java

```diff
@@ -0,0 +1,25 @@
+package sadhana;
+
+//Coding exam_4
+
+public class CommonElementsBitweenArray {
+
+	void printData(int[] arr1, int[] arr2) {
+		for (int i = 0; i < arr1.length; i++) {
+			for (int j = 0; j < arr2.length; j++) {
+				if (arr2[j] == arr1[i])
+					System.out.println(+arr1[i]);
+			}
```

Comment:
you need to break, as once we get match element value then we don't want to check for other element in second array